capture/app.py

```diff
@@ -32,6 +33,27 @@ def main():
         default=False,
         help="print version and exit"
     )
+    parser.add_argument(
+        "-i",
+        "--input-file",
```

Comment:
be consistent and always use underscores in your argument names: change to `input_file`

Reply:
`--input` is arguably a better name